server/service/src/purchase_order_line/insert/mod.rs

```diff
@@ -62,23 +77,33 @@ pub fn insert_purchase_order_line(
             let validate_input = validate::ValidateInput {
                 id: input.id.clone(),
                 purchase_order_id: input.purchase_order_id.clone(),
-                item_id: Some(input.item_id.clone()),
-                item_code: None,
-                // TODO amend default value if we extend standard insert line input
-                requested_pack_size: input.requested_pack_size.unwrap_or_default(), // Default value
+                item_id: input.item_id.clone(),
+                item_code: input.item_code.clone(),
+                requested_pack_size: input.requested_pack_size.unwrap_or_default(),
                 manufacturer_id: input.manufacturer_id.clone(),
             };
             let item = validate(&ctx.store_id.clone(), &validate_input, connection)?;
+
+            let generate_input = InsertPurchaseOrderLineInput {
```

Comment:
Same here... don't need this the item is being returned from validate method, so we use that to get id and code